app/views/api/tags/charges/index.jbuilder

```diff
@@ -1,6 +1,7 @@
 json.array! @charges do |charge|
   json.(charge, :id, :amount, :currency, :charged_back_at, :created_at, :updated_at, :config, :status, :paid, :stripe_url)
   json.display_amount charge.presentation_amount
+  json.display_amount_in_org_currency Charge.presentation_amount(charge.converted_amount(charge.organization.currency), charge.organization.currency)
```

Comment:
Looking at `Charge` code, this conversion operation seems safe from a performance perspective, but given once processed charges don't change over time (is that accurate?) should we use cache for these?

Reply:
Hmm, good question! It seems like it would be pretty safe to cache the charge block with [charge.id, charge.updated_at, charge.organization.updated_at] as the cache key.

Reply:
I think the default ActiveRecord implementation takes into account the `id` and `updated_at` fields, so probably `[charge, charge.organization]` would be enough for the cache key